src/containers/Login/Login.css

```diff
@@ -0,0 +1,10 @@
+
+@media all and (min-width: 480px) {
+    .Login {
+    padding: 60px 0;
+    }
+    .Login form {
+    margin: 0 auto;
+    max-width: 320px;
+    }
+}
```

Comment:
regarding your previous question, all css should be done inline in react
```
<div style={{ padding: '60px' }}>
...
```

Which can be improved by having a styles object outside of the component
```
const styles = {
  container: {
    padding: '60px'
  }
}

...
  <div style={styles.container}>
...
```

But I understand you are allergic to camelCase css attributes, if you would still like to use the classic css attribute, you can try StyledComponents, MaterialUI or emotion. At which I'm more interested to use emotion. But this should be a decision between you and vincent, weigh the pros and cons and preference, since you guys will be using it on day to day basis.